src/Screens/LoginRegistration.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable react/no-unused-state */
+/* eslint-disable react/button-has-type */
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/require-default-props */
+import React, { Component } from 'react';
+import LoginForm from 'Screens/LoginForm';
+import RegisterForm from 'Screens/RegisterForm';
+import PasswordReset from './PasswordReset';
+
+class LoginRegistration extends Component {
+  state = {
+    showLogin: true,
+    showResetPassword: false,
```

Comment:
showLogin: boolean,
showReset: boolean,
showRegistration: boolean,

toggle(formType) => {
     let obj = Object.assign({},  { showLogin: false, showRegistration: false, showReset: false }, {
            [formType]: true,
     })

      this.setState(obj)
}